tsl/test/expected/continuous_aggs_errors.out

```diff
@@ -508,6 +508,62 @@ ERROR:  cannot refresh continuous aggregate on chunk from different hypertable
 DETAIL:  The the continuous aggregate is defined on hypertable "conditions", while chunk is from hypertable "measurements". The continuous aggregate can be refreshed only on a chunk from the same hypertable.
 \set ON_ERROR_STOP 1
 \set VERBOSITY terse
-DROP TABLE measurements;
+-- Add a continuous aggregate on the measurements table and a policy.
```

Comment:
Stating the obvious; would be more useful to describe what is being tested and expected outcome.